packages/next/src/lib/constants.ts

```diff
@@ -88,23 +88,29 @@ export const SERVER_RUNTIME: Record<string, ServerRuntime> = {
 
 const WEBPACK_LAYERS_NAMES = {
   shared: 'sc_shared',
-  server: 'sc_server',
-  client: 'sc_client',
-  action: 'sc_action',
+  /**
+   * React Server Components layer (rsc).
+   */
+  rsc: 'rsc',
+  /**
+   * Server Side Rendering layer (ssr).
+   */
+  ssr: 'ssr',
+  action: 'action',
   api: 'api',
   middleware: 'middleware',
   edgeAsset: 'edge-asset',
-  appClient: 'app-client',
-  metadataRoute: 'app-metadata-route',
+  appPages: 'app-pages',
```

Comment:
Could bring more info to app-pages? Like the description mentioned app browser layer. Because app-pages didn’t indicate it’s server or client?